lib/stat_tracker.rb

```diff
@@ -21,6 +21,39 @@ def initialize(files)
     end
   end
 
+  def highest_scoring_home_team
+    # @game_teams.csv
+    # 1. find total amount of goals scored by each home team and keep track of the amount of games
+    # hash {[team_id #] => [total goals, total games played by home team]}
+    # 2. get hash.values.max
+    # 3. ^find the correlary teamid
+    # iterate thru @teams.csv & match hash.values.max to team_id # 
+      # then output teamName
+
+    # 1.
+    total_goals = @game_teams.each_with_object({}) do |game, hash|
+      # get the curr val for game.team_id or if doesn't exist: set it to [0,0]
+      hash[game.team_id] = hash[game.team_id] || [0, 0]
+      if game.hoa == "home"
+        hash[game.team_id] = [game.goals + hash[game.team_id][0], hash[game.team_id][1] + 1]
+      end
```

Comment:
This is pretty neat, the way I did it, I did two hashes and this reduces that need. I could see this being a helper method that gets reused for the 4 methods we made which are all similar